docs/weekly-questions/joes-water.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Joe's Water Corperation
+sidebar_position: 2
+---
+
+# Joe’s Doubling Points
+
+Joe's Water Corporation is a company that sells bottled water. However, he has competition that also sells bottled water, sometimes at a cheaper price.
+
+If your price is lower, then you get to sell at that store. If not, you lose the chance to sell your water at that store. In the case of a tie, the competition always wins.
+
+You, as the manager of Joe's Water Corporation, can choose to lower the price of water for every single store by M dollars. You should print out 'YES' if it makes you more revenue, or 'NO' if it makes you less revenue. (revenue = sum of prices at stores sold)
```

Comment:
```suggestion
You, as the manager of Joe's Water Corporation, can choose to lower the price of water for every single store you own by $M$ dollars. This could lead to higher revenue (if the discount results in your water being sold at more stores) or lower revenue (if the lower prices outweigh the higher number of locations the water is sold in.) You should print out `YES` if it makes you more revenue, or `NO` if it makes you less revenue.

:::note

The revenue is defined as the sum of the prices at the stores at which your company sells water at a lower price than that of the competitor.

::::
```